### ISSUE TITLE
[1.18.2] Add names on drawer items

### ISSUE THREAD

**Issue by ChimeraGaming** [Buuz135/FunctionalStorage#329]
**Functional Storage Version:**
When items are in drawers, it shows the number but not the name, allow the name to show up with mods like wthaila isis hwyla etc... 

Current mod pack: chroma endless 2

Thanks

**Comment by jerem2772** [comment]
I agree with that, because into big modpacks with lot of items, it's very complicated to use drawer without item's name. Moreover the number is already writen on the drawer

**Comment by ChimeraGaming** [comment]
> This is still an issue in 1.21.1 Modpack: ATM 10

It's not an Issue, It's a [Request]. Not yet Implemented.
